aldryn_people/__init__.py

```diff
@@ -1,2 +1,2 @@
 # -*- coding: utf-8 -*-
-__version__ = '0.4.6'
+__version__ = '0.4.7'
```

Comment:
You really shouldn't be incrementing the version in the same commit as your fix/feature.